playwright-action/helpers/playwrightUtils.js

```diff
@@ -0,0 +1,121 @@
+//@ts-check
+//Playwright configurations for setting up tests
+const { chromium } = require("@playwright/test");
+const path = require('path');
+const browser = await chromium.launch();
+const context = await browser.newContext();
+const page = await context.newPage();
+
+const getFormattedDateTime = () =>{
+      const now = new Date();
+    
+      // Extract date components (YYYY_MM_DD format)
+      const year = now.getFullYear();
+    
+      // Add 1 because months are 0-indexed
+      const month = (now.getMonth() + 1).toString().padStart(2, "0");
+      const day = now.getDate().toString().padStart(2, "0");
+    
+      // Extract time components (HH_MM_SS format)
+      const hours = now.getHours().toString().padStart(2, "0");
+      const minutes = now.getMinutes().toString().padStart(2, "0");
+      const seconds = now.getSeconds().toString().padStart(2, "0");
+    
+      // Concatenate date and time components into the desired format
+      const formattedString = `Meshmap_${year}_${month}_${day}_${hours}_${minutes}_${seconds}.png`;
+    
+      return formattedString;
+};
+
+//utility func to generate css selector and return id and strings
+function waitFor(str) {
+      return `@${str}`;
```

Comment:
I wonder if using @ is recommended.

Reply:
its the same method as ` return "@" + str;`

Reply:
`${}` makes it easier to include variables and expressions within a string without explicit concatenation.

Reply:
what do you think of the login and interceptRequests functions